### ISSUE TITLE
Video element controls keep appearing on each silence skip

### ISSUE THREAD

**Issue by WofWca** [WofWca/jumpcutter#117]
Problem: when using the experimental algorithm (as well as desync correction), when a seek is performed, the controls fade in for ~2 seconds, which may be distracting to people.

It doesn't happen if you remove the controls, so maybe we could add a checkbox that removes them, at least for the local file player:

https://github.com/WofWca/jumpcutter/blob/81b4e507b68d9f7c50e90161326edc65038ae28c/src/entry-points/local-file-player/App.svelte#L89

**Comment by WofWca** [comment]
https://github.com/WofWca/jumpcutter/issues/154 is a clever idea, although I think it's bad for accessibility (e.g. keyboard users) if it's the only thing affecting the presence of `controls`.
But it can make a great combo with what I suggested:

> add a checkbox that removes them, at least for the local file player:

The local file player is a very simple app, so @lowwor, if you want, feel free to make an MR.

**Comment by lowwor** [comment]
Yeah, what I mean is for the local player, most custom video player on the web won't trigger the controls ui.  Indeed, the show and hide of the controls doesn't effect the accessibility, it's the focus problem that forbid the keyboard action which needs to handle separately.

**Comment by WofWca** [comment]
Possible workarounds:
- Don't use the experimental algorithm
- go full screen
- try another in-browser local file player
